src/main/java/sc/fiji/pQCT/selectroi/RoiSelector.java

```diff
@@ -1084,16 +1084,18 @@ else if (result[i + j * width] != 1) {
 		return sleeve;
 	}
 
-	void erode(final byte[] data) {
+	public byte[] erode(byte[] data) {
 		// Erode algorithm
```

Comment:
Why `public`? Does `erode` need to be called outside the pQCT package?

Reply:
No. I think I thought of possibly calling erode from another class but don't think I ever have.

Reply:
Fair enough. Again it's best to be cautious and only make things `public` when you're sure you need access to them.